lib/Common/Memory/Recycler.h

```diff
@@ -1009,7 +1026,7 @@ class Recycler
 
 #ifdef IDLE_DECOMMIT_ENABLED
     HANDLE concurrentIdleDecommitEvent;
-    DWORD needIdleDecommitSignal;
```

Comment:
looks the pal only have LONG version of InterlockedCompareExchange, and there was a compiler error, or need to cast between int32 and long

---
In reply to: [88136723](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r88136723) [](ancestors = 88136723)